app/Core/HttpKernel.php

```diff
@@ -48,7 +48,9 @@ public function handle($request)
             return $e->getResponse();
         } catch (\Throwable $e) {
             if (! app()->make(Environment::class)->debug) {
-                return new RedirectResponse(BASE_URL . "/errors/error500", 201);
+                echo "An error occured";
+                error_log($e);
+                //return new RedirectResponse(BASE_URL . "/errors/error500", 201);
             }
```

Comment:
It would be better if we just checked for the redirect loop here like so:

```php
if (
    ! app()->make(Environment::class)->debug
    && Frontcontroller::getCurrentRoute() !== 'error.error500'
) {
    error_log($e);
    return new RedirectResponse(BASE_URL . "/errors/error500", 201);
}
```